src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -163,6 +163,7 @@ public static void Run(ListSourceArgs args, Func<ILogger> getLogger)
                             getLogger().LogMinimal(string.Format(CultureInfo.CurrentCulture, "{0}{1}", sourcePadding, source.Source));
                         }
 
+                        // Warn for non Https sources
```

Comment:
IMO, comment is redundant because the method name `WarnForHttpSources` is self-explanatory.